packages/blocks/table/src/index.tsx

```diff
@@ -0,0 +1,6 @@
+/**
```

Comment:
This doesn't contain TSX - what prompted the move?

Reply:
I renamed `App.tsx` to `Table.tsx`, and Typescript didn't like importing a `.tsx` file from a `.ts` one, so I renamed the file to satisfy that.

Reply:
But it was already importing a `.tsx` file: `App.tsx`. What is the error you see?